libs/core/langchain_core/messages/tool.py

```diff
@@ -70,6 +71,11 @@ class ToolMessage(BaseMessage):
     .. versionadded:: 0.2.24
     """
 
+    additional_kwargs: dict = Field(default_factory=dict, repr=False)
+    """Currently inherited from BaseMessage, but not used."""
+    response_metadata: dict = Field(default_factory=dict, repr=False)
```

Comment:
i think these could technically be used

Reply:
OK -- we'll address in a follow up PR and update unit tests -- it affects a bunch of snapshots